docs/build/program/apis/cloud.md

```diff
@@ -1332,3 +1332,262 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ResendOrganizationInvite
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Resend a pending organization invite email.
+
+**Parameters:**
+
+- `email` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The email address associated with the invite.
+
+**Returns:**
+
+- None.
+
+**Raises:**
+
+- `GRPCError`: If no pending invite is associated with the provided email address.
+
+```python {class="line-numbers linkable-line-numbers"}
+await cloud.resend_organization_invite("youremail@email.com")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.resend_organization_invite).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetRoverRentalRobots
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Return a list of rover rental robots within an org.
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- [(List[viam.proto.app.RoverRentalRobot])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.RoverRentalRobot): The list of rover rental robots.
+
+```python {class="line-numbers linkable-line-numbers"}
+rental_robots = await cloud.get_rover_rental_robots()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_rover_rental_robots).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### CheckPermissions
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Check validity of a list of permissions.
+
+**Parameters:**
+
+- `permissions` [(List[viam.proto.app.AuthorizedPermissions])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.AuthorizedPermissions): The permissions to validate. For example, `“read_organization”` or `“control_robot”`.
+  See [GitHub](https://github.com/viamrobotics/app/blob/7ddb29e4dcb8cbca6c76c3acaeeb19c544028bd2/auth/models/models.go#L190) for the full list of permissions.
+
+**Returns:**
+
+- [(List[viam.proto.app.AuthorizedPermissions])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.AuthorizedPermissions): The permissions argument, with invalid permissions filtered out.
+
+**Raises:**
+
+- `GRPCError`: If the list of permissions to validate is empty.
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.proto.app import AuthorizedPermissions
+
+permissions = [AuthorizedPermissions(resource_type="organization",
+              resource_id="organization-identifier123",
+              permissions=["control_robot"])]
+
+filtered_permissions = await cloud.check_permissions(permissions)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.check_permissions).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### CreateKey
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Create a new API key.
```

Comment:
Would it make sense to link to a relevant part of the docs about API keys?

Reply:
I can't find one besides the CLI. I think the update to RBAC I'm working on could be a good place to link to in the future.